code/game/jobs/job/medical.dm

```diff
@@ -121,21 +122,21 @@
 	name = "Surgeon"
 	jobtype = /datum/job/doctor
 
-	head = /obj/item/clothing/head/surgery/nt
-	uniform = /obj/item/clothing/under/rank/medical/surgeon/nt
-	suit = /obj/item/clothing/suit/storage/toggle/labcoat/surgeon
+	head = /obj/item/clothing/head/surgery
```

Comment:
Remove this as all it does is fuck up loadout preview and it can be gotten from the locker